file/fs.lisp

```diff
@@ -71,18 +82,36 @@
                (error 'unknown-host
                       :host name
                       :pathname (format nil "~A:" name))))))
-    (t name)))
-
-(defun (setf find-host) (new-value name &optional errorp)
-  (declare (ignore errorp))
+    (file-system-host name)
+    (t (when errorp
+         (error 'mezzano.internals::simple-type-error
```

Comment:
This can just be a normal CL type-error, the message doesn't seem to be adding anything.